rosa_tutorials/rosa-mobb-aws-cloudwatch-publish-metrics.adoc

```diff
@@ -0,0 +1,177 @@
+     > $SCRATCH_DIR/aws-user.json
+----
+
+. Fetch access and secret keys for the IAM user:
++
+[source,terminal]
+----
+$   aws iam create-access-key \
+     --user-name $CLUSTER_NAME-cloud-watch \
+     > $SCRATCH_DIR/aws-access-key.json
+----
+
+. Attach a policy to AWS IAM user:
++
+[source,terminal]
+----
+$   aws iam attach-user-policy \
+     --user-name $CLUSTER_NAME-cloud-watch \
+     --policy-arn "arn:aws:iam::aws:policy/CloudWatchAgentServerPolicy"
+----
+
+== Deploying the CloudWatch Prometheus agent
+
+. Create a namespace for CloudWatch:
++
+[source,terminal]
+----
+$   oc create namespace amazon-cloudwatch
+----
+
+. Download the CloudWatch agent Kubernetes manifests:
++
+[source,terminal]
+----
+$   wget -O $SCRATCH_DIR/cloud-watch.yaml https://mobb.ninja/docs/rosa/metrics-to-cloudwatch-agent/cloud-watch.yaml
+----
+
+. Update the CloudWatch agent Kubernetes manifests:
++
+[source,terminal]
+----
+$   sed -i .bak "s/__cluster_name__/$CLUSTER_NAME/g" $SCRATCH_DIR/cloud-watch.yaml
+$   sed -i .bak "s/__cluster_region__/$CLUSTER_REGION/g" $SCRATCH_DIR/cloud-watch.yaml
+----
+
+. Provide AWS credentials to the CloudWatch agent:
++
+[source,terminal]
+----
+$   AWS_ID=`cat $SCRATCH_DIR/aws-access-key.json | jq -r '.AccessKey.AccessKeyId'`
+$   AWS_KEY=`cat $SCRATCH_DIR/aws-access-key.json | jq -r '.AccessKey.SecretAccessKey'`
+
+$   echo "[AmazonCloudWatchAgent]\naws_access_key_id = $AWS_ID\naws_secret_access_key = $AWS_KEY" \
+     > $SCRATCH_DIR/credentials
+
+$   oc --namespace amazon-cloudwatch \
+     create secret generic aws-credentials \
+     --from-file=credentials=$SCRATCH_DIR/credentials
+----
+
+. Allow the CloudWatch agent to run as a root user inside the container:
++
+[source,terminal]
+----
+$   oc -n amazon-cloudwatch adm policy \
+     add-scc-to-user anyuid -z cwagent-prometheus
+----
+
+. Apply the CloudWatch agent Kubernetes manifests:
++
+[source,terminal]
+----
+$   oc apply -f $SCRATCH_DIR/cloud-watch.yaml
+----
+
+. Check that the pod is running:
++
+[source,terminal]
+----
+$   oc get pods -n amazon-cloudwatch
+----
++
+You should see:
++
+[source,terminal]
+----
+   NAME                                  READY   STATUS    RESTARTS   AGE
+   cwagent-prometheus-54cd498c9c-btmjm   1/1     Running   0          60m
+----
+
+== Creating a sample dashboard
+
+. Download the sample dashboard:
++
+[source,terminal]
+----
+$   wget -O $SCRATCH_DIR/dashboard.json https://raw.githubusercontent.com/rh-mobb/documentation/main/content/docs/rosa/metrics-to-cloudwatch-agent/dashboard.json
+----
+
+. Update the sample dashboard:
++
+[source,terminal]
+----
+$   sed -i .bak "s/__CLUSTER_NAME__/$CLUSTER_NAME/g" $SCRATCH_DIR/dashboard.json
+$   sed -i .bak "s/__REGION_NAME__/$CLUSTER_REGION/g" $SCRATCH_DIR/dashboard.json
+----
+
+. Go to the link:https://us-east-2.console.aws.amazon.com/cloudwatch[CloudWatch section] of the AWS Console.
+
+. Create a dashboard and name it "Kubernetes API Server".
```

Comment:
```suggestion
. Create a dashboard and name it "Kubernetes_API_Server".
```
`Kubernetes API Server` is not a valid dashboard name on AWS. It has validation with `Valid characters in dashboard names include "0-9A-Za-z-_".`